Signal/src/Models/OWSMessagesBubblesSizeCalculator.h

```diff
@@ -1,13 +0,0 @@
-//  Copyright (c) 2016 Open Whisper Systems. All rights reserved.
-
-#import <JSQMessagesViewController/JSQMessagesBubblesSizeCalculator.h>
-
-NS_SWIFT_NAME(MessagesBubblesSizeCalculator)
```

Comment:
Cell sizing is now a responsibility of `ConversationViewItem` and various types of `ConversationViewCell`.